pythonforandroid/bootstraps/common/build/templates/Service.tmpl.java

```diff
@@ -2,10 +2,10 @@
 
 import android.content.Intent;
 import android.content.Context;
-import org.kivy.android.PythonService;
+import org.kivy.android.DashCamPythonService;
```

Comment:
Yeah. I added the reason in gen3-cam-python PR. [Here it is](https://github.com/eroad/gen3-cam-python/pull/9#issue-437763813)
In a nutshell, the PythonService doesn't handle sudden crashes and fails with NPE. Ideally I want to kill the app on error and restart activity so that the service also starts but that seems not feasible. Let me add this comments in the code

Reply:
Comments added in code